app/src/main/java/com/github/se/cyrcle/model/user/User.kt

```diff
@@ -0,0 +1,19 @@
+package com.github.se.cyrcle.model.user
+
+data class User(
+    val userId: String = "",
+    val username: String = "",
+    val firstName: String? = null,
+    val lastName: String? = null,
+    val email: String = "",
+    val profilePictureUrl: String? = null,
+    val homeAddress: String? = null,
+    val favoriteParkingSpots: List<String> = emptyList(),
+    // TODO For some reason these break my serialization/
+    // deserialization and i dont have the time to fix it :
+    // val lastKnownLocation: Location? = null,
+    // val lastLoginTime: Timestamp? = null,
+    // val accountCreationDate: Timestamp? = null,
```

Comment:
Good to have commented these fields for now. We'll have to look into why this happens, but the fields aren't super relevant yet so it's not too big of an issue as of right now